src/components/home/index.js

```diff
@@ -16,9 +19,34 @@ import {
 
 const Home = () => {
   const { showMemberRoleUpdateModal, isSuperUserMode } = userContext();
+  const {
+    state: { newMembers: newMembersList },
+  } = membersContext();
+
   const { query } = useRouter() || { query: { dev: false } };
   const { dev } = query;
   const [isOptionKey, setIsOptionKey] = useState(false);
+  const [newMembers, setNewMembers] = useState([]);
+  const [scrollCount, setScrollCount] = useState(0);
+
+  const membersToShow = 10;
+
+  const fetchNewMembers = () => {
+    const noOfMembersToShow = scrollCount * membersToShow;
+    const slicedNewMembersList = newMembersList.slice(0, noOfMembersToShow);
+    setNewMembers(slicedNewMembersList);
+    setScrollCount(scrollCount + 1);
+  };
+
+  useEffect(() => {
+    fetchNewMembers();
+  }, [newMembersList]);
+
+  const loadMoreData = () => {
+    if (scrollCount > 1) {
```

Comment:
Why do we need the scroll count here?

Reply:
Yeah we don't need scrollCount here. I've removed it